packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMEdge-test.js

```diff
@@ -15,6 +15,10 @@ global.ReadableStream =
 global.TextEncoder = require('util').TextEncoder;
 global.TextDecoder = require('util').TextDecoder;
 
+// Don't wait before processing work on the server.
+// TODO: we can replace this with FlightServer.act().
```

Comment:
Need a better plan for testing "act" things in Flight/Fizz.

Reply:
Ah, jinx